src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/SuggestionDescriptionPanelFromLS.kt

```diff
@@ -54,6 +83,41 @@ class SuggestionDescriptionPanelFromLS(
                 this.add(wrapWithScrollPane(statePanel), BorderLayout.CENTER)
                 SnykBalloonNotificationHelper.showError(unexpectedErrorMessage, null)
             } else {
+                val cefClient = JBCefApp.getInstance().createClient()
```

Comment:
I'm wondering if we should move all this code to its own panel. I imagine this panel will either disappear or become the one that uses JCEF, which is why I started off adding the code in here. But it might be confusing to read so I'm open to suggestions